src/AppMain.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;700;800&display=swap');
+
+div {
```

Comment:
I wouldn't style `<div>`, that's usually bad practice. Make a class for it instead!